### DISCUSSION TITLE
SOC Plotting

### DISCUSSION THREAD

**Discussion by Akila1993** [pybamm-team/PyBaMM#2861] [Q&A]
It is possible to plot SOC variation during a charge or discharge process in DFN model?

**Comment by RuiheLi** [comment]
Hi~ As far as I know, there are no such a variable called "SOC" in DFN. I think the main diffucilties is the definition of SOC. However, you can do this in post-processing:
`sol["Discharge capacity [A.h]"].entries/cap_0`
in which cap_0 is the capacity of the cell you defined, it can be get through a complete discharge step you have run:
`sol["Discharge capacity [A.h]"].entries[-1]-sol["Discharge capacity [A.h]"].entries[0] `
or from the nominal capacity in the parameter set: `para["Nominal cell capacity [A.h]"]`
